arraycontext/impl/pytato/utils.py

```diff
@@ -160,10 +163,7 @@ def map_data_wrapper(self, expr: DataWrapper) -> Array:
 
         # https://github.com/pylint-dev/pylint/issues/3893
         # pylint: disable=unexpected-keyword-arg
-        # type-ignore: discussed at
-        # https://github.com/inducer/arraycontext/pull/289#discussion_r1855523967
-        # possibly related: https://github.com/python/mypy/issues/17375
-        return DataWrapper(  # type: ignore[call-arg]
+        return DataWrapper(
```

Comment:
This works now? What happened?

Reply:
I don't understand it TBH. A few lines down, the symmetric type-ignore is still required. It had something to do with the better `{to,from}_numpy` types. So my guess is that the type of `new_dw` was off. (Which is actually misnamed a bit, it's not a `DataWrapper`---I'll rename it.)

Reply:
Hm.. very weird. It was complaining about the attributes to the constructor, not `new_dw.data`.. Oh well, one less ignore :grin: